src/mcp_atlassian/server.py

```diff
@@ -1169,6 +1203,15 @@ def format_comment(comment: Any) -> dict[str, Any]:
                 "body": comment.get("body"),
             }
 
+        def format_label(label: Any) -> dict[str, Any]:
+            if hasattr(label, "to_simplified_dict"):
+                # Cast the return value to dict[str, Any] to satisfy the type checker
+                return cast(dict[str, Any], label.to_simplified_dict())
+            return {
+                "name": label.get("name", {}),
+                "prefix": comment.get("prefix"),
+            }
+
```

Comment:
The `format_label` helper function has a fallback logic (`return {"name": label.get("name", {}), "prefix": comment.get("prefix")}`) which seems incorrect and unnecessary.

1.  The fallback references `comment.get("prefix")`, but the variable is `label`.
2.  The `get_page_labels` tool call always receives `ConfluenceLabel` model instances from the mixin, which will always have the `to_simplified_dict` method. Therefore, the `if hasattr(label, "to_simplified_dict")` check will always be true, and the fallback path is unreachable.

Consider simplifying this helper to directly use `label.to_simplified_dict()`.

```suggestion
            return cast(dict[str, Any], label.to_simplified_dict())
```